tests/v3_api/test_eks_cluster.py

```diff
@@ -0,0 +1,52 @@
+from .common import *  # NOQA
+
+RANCHER_CLEANUP_CLUSTER = os.environ.get('RANCHER_CLEANUP_CLUSTER', "True")
+ACCESS_KEY = os.environ.get('ACCESS_KEY', "")
+SECRET_KEY = os.environ.get('SECRET_KEY', "")
+
+
+def test_create_eks_cluster():
+
+    if not (ACCESS_KEY and SECRET_KEY):
+        assert (False), "EKS Credentials not provided, cannot create cluster"
```

Comment:
Can this test be skipped if the required environment variables are not present , instead of failing?

Reply:
Skipped